packages/ui-tooltip/src/Tooltip/index.tsx

```diff
@@ -191,15 +191,13 @@ class Tooltip extends Component<TooltipProps> {
         </Trigger>
       )
     } else if (typeof children === 'function') {
-      return children({
+      // TODO: try to remove cast when typing Tooltip
+      return (children as RenderChildren)({
```

Comment:
I'm really not sure why, but TS couldn't correctly calculate the children type, so I used a cast here... maybe it can be sorted out when we fully type Tooltip.